### DISCUSSION TITLE
Best video models on the market?

### DISCUSSION THREAD

**Discussion by BrechtCorbeel** [comfyanonymous/ComfyUI#6229] [General]
I'm speaking Comfyui usable models. I am finally seeing models pop-up left and right and Social media platforms are pushing me more to video so what is your experience here?

I love runway, but their lack of an API and automation makes it a pain to get batches out.

**Comment by BrechtCorbeel** [comment]
I'm currently looking into Hunyuan, seems you can do a lot with little Vram too

**Comment by james-smith-saas** [comment]
Hi ,

"For video models, there are definitely some solid options out there, when you're using tools like ComfyUI.

If you're looking for something that works well with limited VRAM, Hunyuan is a great.

Runway is another popular one, but I feel you on the lack of an API and automation it can be a pain when you need to crank out batches of videos.

LunaBloom is another option for creating text to video generation."
